src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -641,7 +641,7 @@
     <value>None</value>
   </data>
   <data name="Globals_DisableAnimations.Header" xml:space="preserve">

```

Comment:
you can't do this. if I cut a servicing build of 1.6 after you make this change, suddenly the text will change but the logic will not.

Reply:
So, that means that this just can't be a fix that's serviced, right? It'll have to be a part of v1.7?

Reply:
I just put out a servicing update, I'm not going to flip around folks' booleans just days after making them update to 1.6.1041x ;P
What I mean is that changing this for 1.7 will break the key historically for 1.6 builds, BECAUSE it is the same key.

Reply:
So, should we block this PR on renaming this key?

Reply:
I think this should be a part of v1.7 because...
- Settings UI is in preview, so this is the time we _can_ make that change (smaller audience)
- Settings UI is also hidden behind a keybinding users have to opt-in for, so we have an even less reduced number of affected users. Once we include it in defaults.json, we don't have that luxury
- we got explicit feedback from a user saying that this was an issue

We should make sure this change is excluded from any servicing builds prior to v1.7 though. But that's a normal thing to do.